Source/Libraries/Adapters/PhasorProtocolAdapters/CommonPhasorServices.cs

```diff
@@ -1390,6 +1397,35 @@ private static void ValidateStatistics(AdoDataConnection database, Action<string
                     database.Connection.ExecuteNonQuery(string.Format(OutputStreamStatInsertFormat, signalIndex, statName, statDescription, statMethodSuffix, statType, statFormat, signalIndex == 11 ? 1 : 0, statLoadOrder));
                 }
             }
+
+            try
+            {
+                const string QueryDuplicateStatistics = "SELECT SignalReference, COUNT(SignalReference) AS Occurrences FROM Measurement WHERE SignalTypeID = {0} GROUP BY SignalReference HAVING COUNT(SignalReference) > 1";
+                const string DeleteDuplicateStatistics = "DELETE FROM Measurement WHERE SignalReference = {0} AND SignalTypeID = {1} AND PointID NOT IN (SELECT MIN(PointID) FROM Measurement WHERE SignalReference = {0} AND SignalTypeID = {1})";
```

Comment:
Looks like you're using `ExecuteNonQuery<TConnection>(this TConnection connection, ...)` which doesn't accept these format parameters. This `ValidateStatistics()` method uses `string.Format()` for a lot of its queries.

Reply:
Good catch. Not sure why I was referencing `.Connection` there instead of using `AdoDataConnection` wrappers directly - copy/paste I guess, I so rarely use those older functions...